consts.go

```diff
@@ -17,14 +17,14 @@ const (
 )
 
 const (
+	recoveryStateUnclean = iota
+	recoveryStateSwipe
```

Comment:
Swipe

Reply:
Need 4 values:

1. uninitialized (the zero value, just like the other one)
2. unclean
3. wipe
4. clean